nvim-lsp.vim

```diff
@@ -56,13 +60,20 @@ let g:metals_server_version = '0.9.0+18-27d4652a-SNAPSHOT'
       inputBoxProvider             = true;
       quickPickProvider            = true;
       executeClientCommandProvider = true;
+      hoverProvider                = true;
```

Comment:
What is this doing? There is no `hoverProvider` in `InitializationOptions` in Metals.

https://github.com/scalameta/metals/blob/master/metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

Reply:
Strange, I must have copied it from an old config, hoverProvided used to exist, but it's gone from metals for a long time. Deleting